plugins/woocommerce/tests/api-core-tests/tests/customers/customers-crud.test.js

```diff
@@ -15,12 +15,28 @@ const {
  */
 test.describe('Customers API tests: CRUD', () => {
 	let customerId;
+	let testEnvWithSubscriber = false;
+
+	test.beforeAll(async ({
+		request
+	}) => {
+		// Call the API to determine if customer id 2 exists
+		// If so then verify if a subscriber user has been created
+		const response = await request.get('/wp-json/wc/v3/customers/2');
+		const responseJSON = await response.json();
+
+		// if customer id 2 has a subscriber role then set testEnvWithSubscriber 
+		// variable for use in subscriber users tests
+		if (response.status() === 200 && responseJSON.role === 'subscriber') {
+			testEnvWithSubscriber = true;
```

Comment:
Could we update this logic to create the user instead? My only concern is this now skips a test below, so we lose a bit of coverage with that. We should be able to use [this API to create the user](https://developer.wordpress.org/rest-api/reference/users/#create-a-user). In this case, maybe we can update the test to just create the user so it won't be hardcoded to check for a user ID of 2--in case that ever changes.